app/src/androidTest/java/com/invisibleteam/goinvisible/UiDeviceProvider.java

```diff
@@ -0,0 +1,15 @@
+package com.invisibleteam.goinvisible;
+
+import android.support.test.InstrumentationRegistry;
+import android.support.test.uiautomator.UiDevice;
+
+public class UiDeviceProvider {
+    private static UiDevice ourInstance = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation());
```

Comment:
Didn't bothered at all:
- it was automatically generated
- it is private
- whole class is 15 lines of code and no functionality

It is not even a trivial